recipes/semver200/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conan import ConanFile
+from conan.tools.files import get, patch
+from conans.tools import check_min_cppstd
+from conans import CMake
+
+required_conan_version = ">=1.47.0"
+
+class SemVer200Conan(ConanFile):
+    name = "semver200"
+    version = "1.1.0"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zmarko/semver"
+    description = "Semantic versioning for cpp14"
+    topics = ("versioning", "semver", "semantic", "versioning")
+
+    settings = "os", "compiler", "arch", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False, 
+        "fPIC": True,
+    }
+
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, 14)
+
+    def configure(self):
```

Comment:
```suggestion
    def config_options(self):
        if self.settings.os == "Windows":
            del self.options.fPIC

    def configure(self):
```
fPIC option should be removed on Windows.